build-system/tasks/css/index.js

```diff
@@ -140,7 +140,7 @@ function compileCss(options = {}) {
   cssEntryPoints.forEach((entryPoint) => {
     const {path, outJs, outCss, append} = entryPoint;
     promise = promise.then(() =>
-      writeCssEntryPoint(path, outJs, outCss, append)
+      writeCssEntryPoint(path, outJs, outCss, Boolean(append))
```

Comment:
nit: does `append?` work? More dainty IMO

Reply:
I can change the function signatures of `writeCssEntryPoint` and `writeCss` to allow append to be optional, which seems potentially confusing or else I could update `cssEntryPoints` to always include append. I'm opting for the second option to avoid increasing the number of potentially undefined variables.

Reply:
I like the 2nd option too 👍